File: openmp-wg-pII-2016/omp_loop_template.c
//
//

#include <stdio.h>
#include <omp.h>

int main() 
{
//insert pragma omp here
    {
        int i;
        int my_thread = omp_get_thread_num();
//insert pragma for this loop
        for (i=0; i<16; i++)
            printf("Thread %d gets i = %d\n",my_thread,i);
    }
}

File: openmp-wg-pII-2016/omp_loop_template.f90
program omp_loop

   use omp_lib
   integer :: my_thread
!insert omp parallel here. make my_thread a private variable
   my_thread = omp_get_thread_num()
!insert omp for the following loop
   do i=0,15
      print *,"Thread ",my_thread," gets i = ",i
   end do
!close the omp section

!close the outer omp section

end program omp_loop
